src/adservice/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# adsservice
+FROM openjdk:8
+RUN apt-get update && apt-get install net-tools telnet
+WORKDIR /app
+COPY ["build.gradle", "gradlew", "./"]
```

Comment:
I think when you say `./`, it already includes `build.gradle` & `gradlew`.

Also it makes the 

    COPY gradle gradle
    COPY . .

statements below redundant

Reply:
ah okay that's fine I misinterpreted the syntax.